types/frida-gum/frida-gum-tests.ts

```diff
@@ -0,0 +1 @@
+Frida.version; // $ExpectType string
```

Comment:
Consider fleshing this out a bit. :wink:

Reply:
Done 😊